code/model/modelread.cpp

```diff
@@ -4790,52 +4947,55 @@ void model_update_instance(int model_instance_num, int sub_model_num, submodel_i
 	}
 }
 
-void model_do_dumb_rotations_sub(dumb_rotation *dr)
+void model_do_intrinsic_rotations_sub(intrinsic_rotation *ir)
 {
-	polymodel_instance *pmi = model_get_instance(dr->model_instance_num);
+	polymodel_instance *pmi = model_get_instance(ir->model_instance_num);
 	Assert(pmi != nullptr);
 
-	// Handle all submodels which have $dumb_rotate
-	for (auto sub_it = dr->list.begin(); sub_it != dr->list.end(); ++sub_it)
+	// Handle all submodels which have intrinsic rotation
+	for (auto submodel_it = ir->list.begin(); submodel_it != ir->list.end(); ++submodel_it)
 	{
 		polymodel *pm = model_get(pmi->model_num);
-		bsp_info *sm = &pm->submodel[sub_it->submodel_num];
+		bsp_info *sm = &pm->submodel[submodel_it->submodel_num];
 		Assert(pm != nullptr && sm != nullptr);
```

Comment:
Checking `sm != nullptr` isn't really helpful here because in pretty much any case it will be `!= nullptr`.